packages/terra-tag/src/RollupTag.module.scss

```diff
@@ -0,0 +1,100 @@
+// Themes
+@import './clinical-lowlight-theme/RollupTag.module';
+@import './orion-fusion-theme/RollupTag.module';
+
+:local {
+  .rollup-tag-label {
+    font-family: inherit; // Button reset
+    left: 0; // Button reset for IE
+    min-width: 0;
+    overflow: hidden;
+    position: relative; // Button reset for IE
+    text-overflow: ellipsis;
+    top: 0;  // Button reset for IE
+    white-space: nowrap;  
+  }
+
+  @supports (not (-webkit-hyphens:none)) {
+    .rollup-tag-label {
+      top: var(--terra-tags-rollup-tag-label-not-ms-top, -0.01em); // Text alignment for all but IE + Safari
+    }
+  }
+
+  @supports selector(:-moz-is-html) {
+    .rollup-tag-label { 
+      top: var(--terra-tags-rollup-tag-label-moz-top, -0.02em); // Text alignment for Firefox
+    }
+  }
+
+  .rollup-tag:not(.is-active) {
+    background-color: var(--terra-tags-rollup-tag-background-color, rgba(28, 31, 33, 0.04));
+    border: var(--terra-tags-rollup-tag-border, 1px solid rgba(28, 31, 33, 0.1));
+    border-radius: var(--terra-tags-rollup-tag-border-radius, 3px);
+    border-style: solid;
+    border-width: var(--terra-tags-rollup-tag-border-width, 1px);
+    box-shadow: var(--terra-tags-rollup-tag-box-shadow, 0 1px 2px 0 rgba(0, 0, 0, 0.2));
+    box-sizing: border-box;
+    color: var(--terra-tags-rollup-tag-color, #595f62);
+    cursor: pointer;
+    display: inline-block;
+    flex: 1 1 auto;
+    font-family: inherit;
```

Comment:
Is this style necessary?  Isn't this the default behavior?

Reply:
Updated. Thanks
commit: b6dcb40f236e4f0e9b0fde34475a6946d4a2e6ba